packages/fx-core/resource/package.nls.json

```diff
@@ -250,6 +250,7 @@
   "error.cicd.FailedToListMultiEnv": "Failed to list multi env.",
   "error.cicd.PreconditionNotMet": "Some preconditions of inputs are not met.",
   "error.cicd.NoProjectOpened.details": "No project opened.",
+  "plugins.apiconnector.ExecuteUserTask.Message": "Teams Toolkit is going to generate new sample files in your bot or function project. This process will not change your custom code and create the backup files in case revert is needed.",
```

Comment:
Can we change the word to following?
`Teams Toolkit is going to generate new sample file, update package.json and .env.teamsfx.local files in the selected component. The changes will be rollback if unexpected error occurred. Do you want to continue?`
The key thing we want to deliver is which files will be changed. In the meantime, I added a question to the end to let users feel more smooth.

@MuyangAmigo do you have any suggestions on the wording? The messages will be displayed to user before we start updating the project. Just like the confirmation we displayed during provision.

Reply:
When and how will this message be prompted?

Reply:
once user trigger the command, before asking user questions, toolkit will pop up the prompt.
And we need user's approval to continue.